docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -145,8 +145,10 @@ that is printed out.
 For the exact list of named groups ``@linter`` recognizes, see the `API
 documentation <https://api.coala.io/en/latest/>`__.
 
-For more info generally on regexes, see `Python re module
-<https://docs.python.org/3/library/re.html>`_.
+.. note::
```

Comment:
Remove this ```note::``` - Just write it with original style & indentation as was in
```
For more info generally on regexes, see `Python re module
<https://docs.python.org/3/library/re.html>`_.
```
Break the lines at appropriate points so that no line exceeds 80 chars.. 😄

Reply:
Done.